camera-viewer/main.js

```diff
@@ -0,0 +1,17 @@
+'use strict'
+
+const setIpAddress = document.getElementById("ip-address")
```

Comment:
is it set the IP Address? 
Can you explain why you chose `setIpAddress`? 
What do you mean by the word `set`? 

Feel free to explain your idea here before changing and committing

Reply:
I was at a loss with `input`.
The reason I used `setIpAddress` is I felt more natural the sentence` setIpAddress.disabled = true ` than other verbs.
And also `setIpAddress.value`, I decided to consider the combination with the following words.